tests/components/aquacell/test_config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Test the Aquacell config flow."""
+
+import string
+from unittest.mock import AsyncMock, patch
+
+from aioaquacell import ApiException, AuthenticationFailed
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.aquacell.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.components.aquacell import TEST_RESULT_DATA, TEST_USER_INPUT
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def __mock_authenticate(self, user_name, password) -> string:
+    return "refresh-token"
+
+
+async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.FORM
```
Please change this in all your tests